tests/test_flax_auto.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+from transformers import AutoConfig, is_flax_available
+from transformers.testing_utils import require_flax, slow
+
+
+if is_flax_available():
+    from transformers.modeling_flax_auto import MODEL_MAPPING, FlaxAutoModel
+
+
+@require_flax
+class FlaxAutoModelTest(unittest.TestCase):
+    @slow
+    def test_model_from_pretrained(self):
+        for model_config, model_class in MODEL_MAPPING.items():
+
+            for model_name in model_class.pretrained_model_archive_map.keys():
```

Comment:
Is this every used? I think this map is already empty. Are you expected this be filled at some point? I guess I'd like to understand better what `pretrained_model_archive_map` is :-)

Reply:
Recent changes removed this variable, I have updated the unittest behaviour accordingly 👍